Gemfile

```diff
@@ -29,6 +29,8 @@ gem "activerecord-virtual_attributes", "~>1.4.0"
 gem "activerecord-session_store",     "~>1.1"
 gem "acts_as_tree",                   "~>2.7" # acts_as_tree needs to be required so that it loads before ancestry
 gem "ancestry",                       "~>3.0.7",       :require => false
+gem "aws-sdk-ec2",                    "~>1.0",         :require => false # For FileDepotS3
```

Comment:
@Fryguy `FileDepotS3` shouldn't need `ec2` though, right?

Reply:
Okay, this seems wrong:

https://github.com/ManageIQ/manageiq/blob/b5c6cd2f8c96972f1b6b5bd5b816c9b9d5d899fc/app/models/file_depot_s3.rb#L66-L69

Going to investigate...

Reply:
it's possible that the code just used Aws::EC2::Errors, but those errors are available elsewhere (perhaps in Aws::S3)

Reply:
Yeah, I have created a PR to address this here:

https://github.com/ManageIQ/manageiq/pull/19439

Pretty sure the `DynamicErrors` should be defining these under the `Aws::S3` name space, but wouldn't mind a second set of eyes.